notes/feature-script_extensions.md

```diff
@@ -0,0 +1,88 @@
+# Script Extensions
+Support has been added for installing script libraries contained
+in livecode extension packages (.lce files).
+
+## revLoadLibrary and revUnloadLibrary messages
+
+`revLoadLibrary` and `revUnloadLibrary` handlers should be implemented
```

Comment:
@livecodeali I wonder if this is an opportune time to rename rev* stuff?

Reply:
Perhaps yeah... just `loadLibrary`?

Reply:
hmm.... well script extensions might feasibly be something other than libraries I guess. `loadExtension, unloadExtension` would be nice but then it might be expected to appear in `loadedExtensions`?

All having the same prefix might be nice. Perhaps:
- `extensionInitialize`
- `extensionTeardown`
- `extensionStartupScript`